.github/workflows/build-gluon.yml

```diff
@@ -9,8 +9,24 @@ on:
       - master
       - next
       - v20*
+    paths:
+      - "modules"
+      - "Makefile"
+      - "scripts/**"
+      - "package/**"
+      - "patches/**"
+      - "targets/**"
+      - ".github/workflows/build-gluon.yml"
   pull_request:
     types: [opened, synchronize, reopened]
+    paths:
+      - "modules"
+      - "Makefile"
+      - "scripts/**"
+      - "package/**"
+      - "patches/**"
+      - "targets/**"
+      - ".github/workflows/build-gluon.yml"
```

Comment:
This needs to be added to the generator script,

Reply:
Ah, yeah. This was a quick amend after the initial push, since I missed it. And then I did it wrong.

Reply:
Except I did dd it here: https://github.com/freifunk-gluon/gluon/pull/2169/files#diff-c73615de12add5f5e9f1be4721edeaf5999c2e198b04fafd2baf636c92cbcec5. Did you miss that?

Reply:
obviously.